docs/public-lectures.md

```diff
@@ -0,0 +1,38 @@
+The OSG User School is our premier training program for researchers,
+facilitators, and others who are interested in learning more about how to
+use High Throughput Computing to advance research.  This year's program is
```

Comment:
I like typographic quotes, etc., so be sure to check for those (e.g., “year’s”).